library/core/src/cell/once.rs

```diff
@@ -35,7 +41,7 @@ pub struct OnceCell<T> {
 }
 
 impl<T> OnceCell<T> {
-    /// Creates a new empty cell.
+    /// Creates a new cell in the uninitialized state.
```

Comment:
```suggestion
    /// Creates a new uninitialized cell.
```

Slightly less wordy. There are quite a few places that should change "in the (un)initialized state" -> "(un)initialized".